zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -205,10 +205,15 @@ public Factory(WebClient httpClient) {
     final HttpResponse response;
     try (SafeCloseable ignored =
            Clients.withContextCustomizer(ctx -> ctx.logBuilder().name(name))) {
-      HttpRequestWriter httpRequest = HttpRequest.streaming(request.headers());
-      response = httpClient.execute(httpRequest);
-      request.writeBody(httpRequest::tryWrite);
-      httpRequest.close();
+      final RequestHeaders headers = request.headers();
```

Comment:
maybe make a comment as this looks like you have to know that isEndOfStream means a body was already written (it seems like explicit lifecycle control of HTTP/2 stream)

Ex.
```suggestion
      // ElasticsearchStorage healthCheck sets endOfStream to ensure it gets an HTTP response synchronously.
      // The below ensures that flag isn't ignored.
      final RequestHeaders headers = request.headers();
```

Reply:
I realized that we might not need this change. 😅  So I revert it.